src/styles/blocks/articles.css

```
@@ -12,6 +12,110 @@
     font-size: 21px;
 }
 
+@media screen and (max-width: 320px) {
+
+    .main{
+        box-sizing: border-box;
+        display: grid;
+    }
+
+
+    h1{
+        position: relative;
+
+        font-family: RF Dewi;
+        font-size: 21px;
+        line-height: 25px;
+        letter-spacing: 0.03em;
+        color: #FFFFFF;
+
+    }
+
+    h2{
+        font-family: RF Dewi Expanded;
+        font-size: 16px;
+        line-height: 19px;
+        letter-spacing: 0.08em;
+        text-transform: uppercase;
+
+        color: #000000;
+        display: flex;
+        justify-content: flex-start;
+    }
+
+
+    .lead:first-letter{
```

Это псевдоэлемент, лучше их отличать от псевдоклассов двойным двоеточием, как у `::before`, например

```suggestion
    .lead::first-letter{
```